modules.d/01fips-crypto-policies/module-setup.sh

```diff
@@ -22,6 +23,4 @@ installkernel() {
 # called by dracut
 install() {
     inst_hook pre-pivot 01 "$moddir/fips-crypto-policies.sh"
-
-    inst_multiple mount
```

Comment:
Not sure what dracut's policies on duplicate `inst_multiple` calls are — I realize you're removing this because the `fips` module already includes it, but personally, I'd have left it to document the dependency.

Reply:
ok. `base` module also installs `mount`. As I understand there is a slight performance impact as the actual copying of the binary will happen several times.